README.md

```diff
@@ -99,6 +99,29 @@ For work with application, you need to setup your database in docker container.
        docker-compose build --no-cache
        docker-compose up
 
+### Sending mail configurations
+
+1. For sending messages use send_email_async in apps/common/send_email.py
```

Comment:
not a good idea to put in documentation file names bcs u can change names, but forgot to update docs
so pls try to avoid it

Reply:
fixed